yaml.go

```diff
@@ -136,6 +136,9 @@ func decode(reader io.Reader) ([]unstructured.Unstructured, error) {
 
 // isURL checks whether or not the given path parses as a URL.
 func isURL(pathname string) bool {
+	if _, err := os.Lstat(pathname); err == nil {
```

Comment:
Is this necessary on Windows or something?

Reply:
Without this, the "absolute path" test in `yaml_test.go` will try to interpret "c:\Users\...." as a URL with `Scheme = "c"` and some other garbage.

I can drop this if you want; I'm not sure what the best way to distinguish between file paths on Windows + Unix and URIs would be.

Reply:
It's fine. I was just curious.